lib/digital_ocean/ssh_keys.ex

```diff
@@ -0,0 +1,96 @@
+defmodule DigitalOcean.SSHKeys do
```

Comment:
Resource modules should be singular. Could you please rename to `SSHKey`? Also, could you please rename the file to `ssh_key.ex`?